app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @user = User.all
+  end
+
+  def display
+    @user = User.find(params[:user_id])
+  end
```

Comment:
- [ ] Based on the requirements we should have a `#show` action not `#display`. Kindly fix that. It is also a good idea to exhaust the inbuilt route actions provided by Rails before creating similar ones